### PR TITLE
feat: refactor options

### PR DESCRIPTION
**Pull request by notdryft**
See https://github.com/gatling/gatling-enterprise-plugin-commons/pull/149 for the corresponding changes in the enterprise-commons library

### CODE REVIEW

File: build.sbt
Comment (notdryft) [inline build.sbt:12]: Waiting for release before merging

**Review by slandelle — APPROVED** [review]

File: src/main/scala/io/gatling/sbt/settings/gatling/EnterprisePluginTask.scala
Comment (guilgaly) [inline src/main/scala/io/gatling/sbt/settings/gatling/EnterprisePluginTask.scala:49]: Dead doc link, we could fix it too

**Review by guilgaly — APPROVED** [review]

**Pull request merged**
